.github/workflows/ci.yml

```diff
@@ -114,44 +135,55 @@ jobs:
       - uses: Swatinem/rust-cache@a95ba195448af2da9b00fb742d14ffaaf3c21f43
         with:
           prefix-key: "0" # change this to invalidate CI cache
+          shared-key: "cargo_nextest"
           save-if: "false" # use the cache from nextest, but don’t double-save
-      - run: cargo build -p neard --bin neard --features nightly
-      - name: run spin_up_cluster.py
-        # Note: We're not running spin_up_cluster.py for non-nightly
-        # because spinning up non-nightly clusters is already covered
-        # by other steps in the CI, e.g. upgradable.
-        run: |
-          cd pytest
-          python3 -m pip install --progress-bar off --user -r requirements.txt
-          python3 tests/sanity/spin_up_cluster.py
+      - run: pip3 install --user -r pytest/requirements.txt
+      - run: cargo build --profile quick-release -p neard --bin neard --features nightly
```

Comment:
Nit: add a comment that download-artifact doesn’t work because it doesn’t build with nightly features